projects/canopy/src/lib/side-nav/side-nav-bar-item/side-nav-bar-item.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component, HostBinding, Input, ViewEncapsulation } from '@angular/core';
+
+@Component({
+  selector: 'lg-side-nav-bar-item',
+  templateUrl: './side-nav-bar-item.component.html',
+  styleUrls: ['./side-nav-bar-item.component.scss'],
+  encapsulation: ViewEncapsulation.None,
```

Comment:
Most of the components should probably be switched to `changeDetection: ChangeDetectionStrategy.OnPush`

Reply:
Fixed.